test/e2e/run_test.go

```diff
@@ -61,6 +62,70 @@ var _ = Describe("Podman run", func() {
 		Expect(match).Should(BeTrue())
 	})
 
+	It("podman run selinux disable test", func() {
+		if !selinux.GetEnabled() {
+			Skip("SELinux not enabled")
+		}
+		session := podmanTest.Podman([]string{"run", "-it", "--security-opt", "label=disable", ALPINE, "cat", "/proc/self/attr/current"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match, _ := session.GrepString("unconfined_t")
+		Expect(match).Should(BeTrue())
+	})
+
+	It("podman run selinux type check test", func() {
+		if !selinux.GetEnabled() {
+			Skip("SELinux not enabled")
+		}
+		session := podmanTest.Podman([]string{"run", "-it", ALPINE, "cat", "/proc/self/attr/current"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match1, _ := session.GrepString("container_t")
+		match2, _ := session.GrepString("svirt_lxc_net_t")
+		Expect(match1 || match2).Should(BeTrue())
+	})
+
+	It("podman run selinux type setup test", func() {
+		if !selinux.GetEnabled() {
+			Skip("SELinux not enabled")
+		}
+		session := podmanTest.Podman([]string{"run", "-it", "--security-opt", "label=type:spc_t", ALPINE, "cat", "/proc/self/attr/current"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match, _ := session.GrepString("spc_t")
+		Expect(match).Should(BeTrue())
+	})
+
+	It("podman run seccomp undefine test", func() {
```

Comment:
This and the other Seccomp test shouldn't required the `selinux.GetEnabled()` check - Seccomp should work without SELinux.

Reply:
OK, will remove this. Thanks a lot.